.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       with:
           go-version-file: 'tools/go.mod'
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@ec5d18412c0aeab7936cb16880d708ba2a64e1ae
+      uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd
```

Comment:
optional: comment that dependabot will keep up to date:

`uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd # v7.0.0`

Reply:
I haven't got these comments in other places so I will leave them out here.